packages/window-size-tool/rules.json

```
@@ -33,8 +33,7 @@
   {
     "width": "> 576 and <= 768",
     "terms": [
-      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//CLUSTER) > 0",
-      "count(//LIST[count(FIELD | CONTAINER) <= 2]) > 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0"
+      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//CLUSTER) > 0 and count(//LIST[count(FIELD | CONTAINER) <= 2]) > 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0"
```

This will now only apply when the modal contains both a cluster _and_ a list. We need to cater for when there might only be a cluster or only a list.

```suggestion
      "count(//CLUSTER[@NUM_COLS > 1 ]) = 0 and count(//LIST[count(FIELD | CONTAINER) > 2]) = 0"
```